Sprint-3/todo-list/script.js

```diff
@@ -1,25 +1,72 @@
-function populateTodoList(todos) {
-  let list = document.getElementById("todo-list");
-  // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
-}
+const removeAllButton = document.getElementById("remove-all-completed");
+const form = document.getElementById("todo-form");
+const input = document.querySelector(".input");
+const todoList = document.getElementById("todo-list");
 
-// These are the same todos that currently display in the HTML
-// You will want to remove the ones in the current HTML after you have created them using JavaScript
 let todos = [
   { task: "Wash the dishes", completed: false },
   { task: "Do the shopping", completed: false },
 ];
 
-populateTodoList(todos);
+// Populate the todo list with initial tasks
+function populateTodoList(todos) {
+  todos.forEach((item) => {
+    addTodoToDOM(item.task, item.completed);
+  });
+}
+
+// Add todo item to the DOM
+function addTodoToDOM(task, completed = false) {
+  const newTodo = document.createElement("li");
+  newTodo.classList.add("todo-item");
+  if (completed) newTodo.classList.add("done");
```

Comment:
This works, but a lot of people always use `{}` around the body of things like `if` statements. You _need_ to do this if you have more than one statement in the body, but a lot of people always do. Have a read of https://www.blackduck.com/blog/understanding-apple-goto-fail-vulnerability-2.html for an example of why.

Reply:
good point, thanks)